src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Views/Dashboard/InModerationApplications.cshtml

```diff
@@ -73,11 +74,11 @@
                                             }
                                             else if (item.ModerationStatus == ModerationStatus.InProgress)
                                             {
-                                                <strong class="das-tag das-tag">In moderation</strong>
+                                                <strong class="das-tag">In moderation</strong>
```

Comment:
Just want to check if just `das-tag` is okay here? In the dashboard we've also got `das-no-wrap`